tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -187,7 +194,8 @@ def _forward_backward_test_impl(
                 deallocate_pipeline_output=deallocate_pipeline_outputs,
             )
 
-            if dtype == torch.double:
+            if dtype == get_dtype_for_comparison():
+                torch.cuda.synchronize()
```

Comment:
QQ: I think this would let us run numerical checks in single precision if device prior to Ampere.
Is this expected?